grammars/perl 6.cson

```diff
@@ -0,0 +1,305 @@
+fileTypes: [
+  "p6"
+  "pl"
```

Comment:
Sure, I can remove the common ones for now.

It could be possible to detect 'use v6' and then say the rest of the file is considered 'source.perl6' instead of 
'source.perl'. 

For now, I'll just remove the common file names from the list though.